src/application/imu_handler/imu_handler.h

```diff
@@ -1,8 +1,26 @@
 #ifndef IMU_HANDLER_H
 #define IMU_HANDLER_H
 
+#include "rocketlib/include/common.h"
+
+/**
+ * @brief Initialize the IMU handler module
+ * Must be called before scheduler starts
+ * @return Status of initialization
+ */
+w_status_t imu_handler_init(void);
+
+/**
+ * @brief Execute one iteration of the IMU handler processing
+ * Reads data from all IMUs and updates the estimator
+ * @return Status of the execution
+ */
+w_status_t imu_handler_run(void);
```

Comment:
I was going back and forth w @celery6 on this. I don't think we should expose functions like this in headers, because they can cause unintended behaviour if run outside the intended task.

Either wrap in `#ifdef GTEST` so it is only included in the header for unit testing, or pass static members as parameters instead of accessing them from the outer scope. So like `w_status_t imu_handler_run(imu_handler_state_t *state);`. I tend to prefer the latter but in this case it also affects state est so ifdefs might be more reliable.